content/zh/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -166,52 +174,52 @@ minikube service example-service --url
-external traffic to node-local or cluster-wide endpoints. There are two available
-options: Cluster (default) and Local. Cluster obscures the client source
-IP and may cause a second hop to another node, but should have good overall
-load-spreading. Local preserves the client source IP and avoids a second hop
-for LoadBalancer and NodePort type services, but risks potentially imbalanced
-traffic spreading.
+* `.spec.externalTrafficPolicy` - denotes if this Service desires to route
+  external traffic to node-local or cluster-wide endpoints. There are two available
+  options: `Cluster` (default) and `Local`. `Cluster` obscures the client source
+  IP and may cause a second hop to another node, but should have good overall
+  load-spreading. `Local` preserves the client source IP and avoids a second hop
+  for LoadBalancer and NodePort type Services, but risks potentially imbalanced
+  traffic spreading.
 -->
-* `service.spec.externalTrafficPolicy` - 表示此服务是否希望将外部流量路由到节点本地或集群范围的端点。
-  有两个可用选项：Cluster（默认）和 Local。
-  Cluster 隐藏了客户端源 IP，可能导致第二跳到另一个节点，但具有良好的整体负载分布。
-  Local 保留客户端源 IP 并避免 LoadBalancer 和 NodePort 类型服务的第二跳，
+* `.spec.externalTrafficPolicy` - 表示此 Service 是否希望将外部流量路由到节点本地或集群范围的端点。
+  有两个可用选项：`Cluster`（默认）和 `Local`。
+  `Cluster` 隐藏了客户端源 IP，可能导致第二跳到另一个节点，但具有良好的整体负载分布。
+  `Local` 保留客户端源 IP 并避免 LoadBalancer 和 NodePort 类型服务的第二跳，
   但存在潜在的不均衡流量传播风险。
 
 <!--
-* `service.spec.healthCheckNodePort` - specifies the health check nodePort
-(numeric port number) for the service. If `healthCheckNodePort` isn't specified,
-the service controller allocates a port from your cluster's NodePort range. You
-can configure that range by setting an API server command line option,
-`--service-node-port-range`. It will use the
-user-specified `healthCheckNodePort` value if specified by the client. It only has an
-effect when `type` is set to LoadBalancer and `externalTrafficPolicy` is set
-to Local.
+* `.spec.healthCheckNodePort` - specifies the health check node port
+  (numeric port number) for the service. If you don't specify
+  `healthCheckNodePort`, the service controller allocates a port from your
+  cluster's NodePort range.
+  You can configure that range by setting an API server command line option,
+  `--service-node-port-range`. The Service will use the user-specified
+  `healthCheckNodePort` value if you specify it, provided that the
+  Service `type` is set to LoadBalancer and `externalTrafficPolicy` is set
+  to `Local`.
 -->
-* `service.spec.healthCheckNodePort` - 指定服务的 healthcheck nodePort（数字端口号）。
-  如果未指定 `healthCheckNodePort`，服务控制器从集群的 NodePort 范围内分配一个端口。
+* `.spec.healthCheckNodePort` - 指定服务的 healthcheck nodePort（数字端口号）。
+  如果你未指定 `healthCheckNodePort`，服务控制器从集群的 NodePort 范围内分配一个端口。
   你可以通过设置 API 服务器的命令行选项 `--service-node-port-range` 来配置上述范围。
-  它将会使用用户指定的 `healthCheckNodePort` 值（如果被客户端指定）。
-  仅当 `type`  设置为 LoadBalancer 并且 `externalTrafficPolicy` 设置为 Local 时才生效。
+  在服务 `type` 设置为 LoadBalancer 并且 `externalTrafficPolicy` 设置为 Local 时，
```

Comment:
nit: `Local` should be `` `Local` `` according to English content.

Reply:
Done, thanks.